anchore_engine/services/catalog/image_content/get_image_content.py

```diff
@@ -0,0 +1,152 @@
+import base64
+import json
+
+from anchore_engine.services.apiext.api import helpers
+
+import anchore_engine
+from anchore_engine import utils, db
+from anchore_engine.apis.exceptions import ResourceNotFound, BadRequest
+from anchore_engine.common.helpers import make_anchore_exception
+from anchore_engine.db import db_catalog_image
+from anchore_engine.subsys import taskstate, logger
+
+
+class ImageContentGetter:
```

Comment:
Good approach for abstraction here. Enables future use of caches, etc easily (e.g. CachingImageContentGetter...)